examples/mobile/UIComponents/components/card/link.html

```diff
@@ -6,6 +6,7 @@
 	<meta content="width=device-width, user-scalable=no" name="viewport" />
 	<link href="../../lib/tau/mobile/theme/changeable/tau.css" rel="stylesheet" />
 	<link href="../../css/style.css" rel="stylesheet" />
+	<script src="https://unpkg.com/mustache@latest"></script>
```

Comment:
For using the webclip in the device home, is it also included <script src="https://unpkg.com/mustache@latest"></script>?

Reply:
Yes, but it is the first working version. This library can me merge with tau or included to tau/libs.